packages/client/src/rpc/modules/engine/engine.ts

```diff
@@ -1513,4 +1521,13 @@ export class Engine {
     }
     return payloads
   }
+
+  private async getBlobsV1(params: [[Bytes32]]): Promise<(BlobAndProofV1 | null)[]> {
+    const blobsAndProof: (BlobAndProofV1 | null)[] = []
```

Comment:
Should we put a check here which throws if `params.length` is too long?

Reply:
yea we could add